src/Framework/PaymentGateways/Commands/SubscriptionComplete.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace Give\Framework\PaymentGateways\Commands;
+/***
+ * @unreleased
+ */
+class SubscriptionComplete implements GatewayCommand {
+    /**
+     * @var string
+     */
+    public $transactionId;
+    /**
+     * @var string
+     */
+    public $profileId;
```

Comment:
Let's make sure to document the Command properties well to be clearer to 3rd parties (and us). Correct me if I'm wrong, @ravinderk, but I believe the `transactionId` is the charge ID as unique on the gateway; the profile ID is the URL to view the transaction on the gateway site.

Honestly, these are legacy terms, so it would even be useful to just rename these here and slowly start to change the names elsewhere.

Reply:
@JasonTheAdams Profile ID is a subscription id in the payment gateway system. The Profile ID is linked to the payment gateway subscription details page for admin convenience on the subscription details page.

Reply:
Oh. Well there you go. 😆

Reply:
Okay so I decided to rename these to `gatewayTransactionId` and `gatewaySubscriptionId`.  It makes it much clearer that it's specific to the gateway and not give - especially when there is another `subscriptionId` in the handler that is a give subscription ID not a gateway subscription ID. 81bae27b

Reply:
Love it! 🙌